Directory.Build.targets

```diff
@@ -24,44 +24,44 @@
-    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.3.0" />
-    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.6.0" />
+    <PackageReference Update="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.2.2" />
+    <PackageReference Update="Azure.Identity" Version="1.7.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.4.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.4.0" />
 
-    <PackageReference Update="Serilog" Version="2.10.0" />
+    <PackageReference Update="Serilog" Version="2.12.0" />
     <PackageReference Update="Serilog.Extensions.Logging" Version="3.1.0" />
     <PackageReference Update="Serilog.Formatting.Compact" Version="1.1.0" />
-    <PackageReference Update="Serilog.Settings.Configuration" Version="3.3.0" />
-    <PackageReference Update="Serilog.Sinks.Console" Version="4.0.1" />
-    <PackageReference Update="Serilog.Sinks.Seq" Version="5.1.1" />
+    <PackageReference Update="Serilog.Settings.Configuration" Version="3.4.0" />
+    <PackageReference Update="Serilog.Sinks.Console" Version="4.1.0" />
+    <PackageReference Update="Serilog.Sinks.Seq" Version="5.2.1" />
 
     <PackageReference Update="IdentityModel" Version="6.0.0" />
     <PackageReference Update="Microsoft.IdentityModel.Tokens" Version="$(SystemIdentityModelVersion)" />
     <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="$(SystemIdentityModelVersion)" />
 
     <PackageReference Update="FirebaseAdmin" Version="2.3.0" />
-    <PackageReference Update="Google.Cloud.Firestore" Version="2.5.0" />
+    <PackageReference Update="Google.Cloud.Firestore" Version="3.0.0" />
 
-    <PackageReference Update="Microsoft.Data.SqlClient" Version="4.1.0" />
+    <PackageReference Update="Microsoft.Data.SqlClient" Version="5.0.1" />
 
     <PackageReference Update="Cronos" Version="0.7.1" />
     <PackageReference Update="Dapper" Version="2.0.123" />
     <PackageReference Update="FluentValidation" Version="9.5.4" />
     <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Update="Polly" Version="7.2.3" />
-    <PackageReference Update="Sendgrid" Version="9.27.0" />
+    <PackageReference Update="Sendgrid" Version="9.28.1" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Update="Microsoft.AspNetCore.Authentication.JwtBearer" Version="$(AspNetCoreVersion)" />
-    <PackageReference Update="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
     <PackageReference Update="Microsoft.AspNetCore.Mvc.Testing" Version="$(AspNetCoreVersion)" />
     <PackageReference Update="Microsoft.AspNetCore.Razor.Language" Version="$(AspNetCoreVersion)" />
+    <PackageReference Update="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
```

Comment:
Why is this one our of order now?

Reply:
Gah, moved it accidentally. Will rollback.